modular-docs-manual/content/topics/module_guidelines-reference.adoc

```diff
@@ -20,3 +20,34 @@ To make the reference data easier to scan, organize it in a logical order (such
 * link:http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#tables[Tables]
 
 If you have a large volume of the same type of information to document, use a structure into which the information details can fit, and then document each logical unit of information as one reference module. For example, think of man pages, which document very different information details, but which still use consistent titles and formats to present those details in a uniform information structure.
+
+If the reference module is large and complex, consider splitting the reference module into multiple standalone reference modules. If you cannot split the module into meaningful standalone modules, consider using subheadings in the module to structure the content for improved reader navigation. If you use subheadings in a reference module, you can add the `[discrete]` tag to exclude each subheading from the table of contents, if needed. However, in many cases, subheadings are helpful to include in the table of contents to improve content searchability.
+
+The following examples illustrate a discrete and a standard subheading:
+
+.Example discrete subheading excluded from table of contents
+[source]
+----
+= My reference module
+
+Reference introduction and body
+
+[discrete]
+== My reference module subheading
+
+More reference body content
+----
+
+.Example subheading included in table of contents
+[source]
+----
+= My reference module
+
+Reference introduction and body
+
+== My reference module subheading
+
+More reference body content
+----
+
+NOTE: You can use subheadings in concept or reference modules, but not in procedure modules.
```

Comment:
Why not?

Reply:
This was explicitly requested when mod docs was first kicked off by Christy and Sheldon, where subheadings were fine for cons and refs but not for procs. The rationale was/is, given the nature of procs and their need to be centered on a clear, focused task, if you need to divide a proc into different subheadings, then it should almost always (or simply always) be separate procedures. Cons and refs don't have the same stringency because concepts and references can require a lot of content but still be a part of the same overall idea that might not be wise to split up into sub-modules. That's highly subjective of course, as we discussed with Don in this thread, and there is surely an edge case for a multi-headed procedure, but to date, the recommendation from the beginning has been to permit subheads in cons and refs but not procs. So what my PR does is simply clarity that, not add anything new or change anything that wasn't already the case from the beginning.

Now, if we're having second thoughts, we should discuss as a team in that case. If this simply opens a can of worms, I can always revert this PR altogether. But this comes up repeatedly with those I work with and what writers are doing to address this need is overly divide content into artificial and overly minute con and ref modules simply in order to avoid using subheadings, which isn't necessary.